device_backends/LogicalNameMapping/include/LNMAccessorPlugin.h

```diff
@@ -275,10 +275,16 @@ namespace ChimeraTK { namespace LNMBackend {
 
     // obtain desired target type from plugin implementation
     auto type = getTargetDataType(typeid(UserType));
+    auto info = _info.lock();
```

Comment:
Yes, I was wondering the same. Bit there is no explicit test needed here. This is just locking a weak pointer. If that fails the further access to the pointer would result in a nulltpr exception.

Reply:
Ah, I see what you mean. This is not in a test, it's in the code and the parent should have gone, so you cannot lock the weak pointer. I will check that.

Reply:
I will not do this now to avoid unnecessary merge conflicts. I guess this is one of the places where Martin H.'s new deep-copy mechanism will prevent the weak pointer and use a clone() function to duplicate this accessor (?), so I am expecting this to be touched anyway.